sql/branch.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
+    UPDATE payment_instructions SET due = 0 WHERE due > 9.41;
```

Comment:
This means that anyone with dues and a failing credit card is getting a big reset. I think that means we're leaving a little money on the table. An alternative would be to reset due to `9.41`, and then if/when they fix their card, we'd charge them that much instead of having to build back up. Right?

Was that the original intention at https://github.com/gratipay/gratipay.com/pull/3876#discussion-diff-51040424?